chromium_src/chrome/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/customize_toolbar_handler.cc

```diff
@@ -6,14 +6,25 @@
 #include "chrome/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/customize_toolbar_handler.h"
 
 #include "base/memory/raw_ref.h"
+#include "base/notreached.h"
 #include "brave/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/brave_action.h"
 #include "brave/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/list_action_modifiers.h"
 
+// This is actually layering violation but the original file includes this.
```

Comment:
I think this is not a violation any more.
In the https://chromium.googlesource.com/chromium/src/+/main/docs/chrome_browser_design_principles.md,
```
The historical rule restricting access to views in //chrome/browser and //chrome/browser/ui has been removed.
The historical rule disallowing ui code in //chrome/browser has been removed.
```

Reply:
Oh, didn't know that. Thanks. Removed the comment